app/controllers/organization_wizzard_controller.rb

```diff
@@ -0,0 +1,77 @@
+class OrganizationWizzardController < ApplicationController
+  layout 'settings'
+  before_action :set_organization
+  before_action :set_organization_wizzard
+  before_action :redirect_if_complite_second_step, only: [ :new_account,
```

Comment:
@dissident "complete", not "complite". And this name is too long. Yo may do it short. Something like `require_second_step`